client/src/entrypoints/admin/icons.js

```diff
@@ -0,0 +1,5 @@
+import { initIconSprite } from '../../includes/initIconSprite';
+
+const url = document.currentScript.dataset.iconUrl;
+const container = document.querySelector('[data-sprite]');
+initIconSprite(container, url);
```

Comment:
```suggestion

initIconSprite(container, url);
```

A little bit of whitespace can help readability, I will add a line break here.